packages/api/schema/stryker-core.json

```diff
@@ -191,7 +190,21 @@
               "type": "string"
             }
           ],
-          "default": "**/*+(.js|.ts|.cjs|.mjs)?(x)"
+          "default": false
+        },
+        "disableTypeChecking": {
+          "description": "Configure a pattern that matches the files of which type checking has to be disabled. This is needed because Stryker will create (typescript) type errors when inserting the mutants in your code (by design). Stryker disables type checking by inserting `// @ts-nocheck` atop those files and removing other `// @ts-xxx` directives (so they won't interfere with `@ts-nocheck`). The default setting allows these directives to be stripped from all JavaScript and friend files in your sandbox, you can specify a different glob expression or set it to `false` to completely disable this behavior.",
+          "anyOf": [
+            {
+              "enum": [
+                false
+              ]
+            },
+            {
+              "type": "string"
+            }
+          ],
+          "default": "**/*+(.js|.ts)?(x)"
```

Comment:
Simplify - ???

```suggestion
          "default": "**/*.{js,jsx,ts,tsx}?(x)"
```

Reply:
I've changed it to `**/*.{js,ts,jsx,tsx,html,vue}`